third_party/bazel/mkldnn/mkl_include.BUILD

```diff
@@ -0,0 +1,3 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(["include"])
```

Comment:
is it possible to expose those as header-only targets instead of files? So does static libs.